components/esp_wifi_remote/examples/mqtt/sdkconfig.defaults

```diff
@@ -1,3 +1,4 @@
+CONFIG_ESP_WIFI_REMOTE_LIBRARY_EPPP=y
```

Comment:
do we still wish to use eppp as default solution here?

Reply:
I would still keep `eppp` as default for this example (since I wanted to build it with wifi capable chips as well, which doesn't seem to work with esp-hosted)